Rakefile

```diff
@@ -14,6 +14,10 @@ namespace :notify do
     "https://docs.payments.service.gov.uk/api/pages.json",
   ]
 
+  limits = {
+    "https://dcs-service-manual.cloudapps.digital/api/pages.json" => 5
```

Comment:
Just to double check this means that only the DCS Service Manual gets the paging option?

Reply:
Indeed - I wouldn't want to enforce this on any other team without their permission, and their needs may be different than hours.

Reply:
Ah I approved it but again I'm not a person with merging rights 🤷

Reply:
Gah, thanks anyway. I've requested a review from one of the teams that "owns" the repo.

Reply:
🤷🏻‍♂️  Looks like a github misconfiguration - the team that owns the repo only has a few people in it.

I've added the GOV.UK teams, so Kelvin's approval counts now. There are quite a few people on that team who can administer this repo now, so if we need to add more teams in the future we can.